### PR TITLE
feat(icons): add new shield Icon

### PR DESCRIPTION
**Pull request by Sunyang730**
Screen Shot 2019-05-23 at 1 51 11 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @Sunyang730 has signed the CLA. Thanks for the pull request!

**Comment by booc0mtaco** [comment]
@Sunyang730 can we get an export of this icon such that the view box and default sizes are 32x32? for icons that aren't perfectly square, they should be  centered in the view box.

File: src/icons/general/IconShield.js [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
+            <path
+                fill={white}
+                stroke={color}
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth="2"
+                d="M1 5.615S13.75 12.923 26.5 1C39.25 12.923 52 5.615 52 5.615v40.962L26.5 61 1 46.577V5.615z"
+            />
+            <path fill={color} fillOpacity={opacity} d="M26 7c-10.5 9.538-21 3.692-21 3.692v32.77L26 55V7z" />
+            <path stroke={color} strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M44 19v20" />
```
Comment (priyajeet) [inline src/icons/general/IconShield.js]: className `fill-color` and `stroke-color` respectively that can change

File: src/icons/general/IconShield.js [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
 
-const IconShield = ({ className = '', color = '#888888', height = 14, title, width = 12 }: Props) => (
+const IconShield = ({ className, color = boxBlue, height = 62, opacity = 0.2, width = 53, title }: Props) => (
     <AccessibleSVG
-        className={`icon-governance ${className}`}
+        className={classNames('bdl-IconShield', className)}
         height={height}
         title={title}
-        viewBox="0 0 12 14"
+        viewBox="0 0 53 62"
```
Comment (priyajeet) [inline src/icons/general/IconShield.js]: See if this can be made 32x32

File: src/icons/general/IconShield.js [hunk 3]
```diff
@@ -1,34 +1,51 @@
...
-        <g
-            className="stroke-color"
-            fill="none"
-            fillRule="evenodd"
-            stroke={color}
-            strokeLinecap="round"
-            strokeLinejoin="round"
-        >
-            <path d="M.5 1.5S3.25 3.08 6 .5c2.75 2.58 5.5 1 5.5 1v8.88L6 13.5.5 10.37V1.5zM9.5 4.5v4" />
+        <g fill="none" fillRule="evenodd">
```
Comment (priyajeet) [inline src/icons/general/IconShield.js]: `<g>` needed?

File: src/icons/general/IconShield.js [hunk 4]
```diff
@@ -1,34 +1,51 @@
...
+            <path
+                fill={white}
+                stroke={color}
+                strokeLinecap="round"
```
Comment (priyajeet) [inline src/icons/general/IconShield.js]: className stroke-color needed here?

**Comment by Sunyang730** [comment]
Updated the PR to keep it with the standard of 32 x 32 icon. Will do override of the style within the component that it's going to use to reflect something that match the mock up.

**Comment by Sunyang730** [comment]
Screen Shot 2019-06-03 at 3 17 47 PM

**Pull request merged**